File: 10817/10817.cpp
#include <iostream>
using namespace std;
int main() {
	int n1 = 0;
	int n2 = 0;
	int num[3];
	cin >> num[0] >> num[1] >> num[2];
	for (int i = 0; i < 3; i++) {
		if (num[i] > n1) {
			n2 = n1;
			n1 = num[i];
		}
		else if (num[i] > n2)
			n2 = num[i];
	}
	cout << n2;
	return 0;
}

File: 10817/10817.py
n1 = 0
n2 = 0
num = list(map(int, input().split(' ')))
for i in range(0, 3):
    if num[i] > n1:
        n2 = n1
        n1 = num[i]
    elif num[i] > n2:
        n2 = num[i]
print(n2)